src/MixedModels.jl

```diff
@@ -1,35 +1,51 @@
 module MixedModels
 
-using Arrow
-using BSplineKit
-using DataAPI
-using Distributions
-using GLM
-using JSON3
-using LinearAlgebra
-using Markdown
-using NLopt
-using Random
-using PooledArrays
-using ProgressMeter
-using SparseArrays
-using StaticArrays
-using Statistics
-using StatsAPI
-using StatsBase
-using StatsModels
-using StructTypes
-using Tables
-using TypedTables
-
-using LinearAlgebra: BlasFloat, BlasReal, HermOrSym, PosDefException, copytri!
+using Arrow: Arrow
 using Base: Ryu, require_one_based_indexing
-using DataAPI: levels, refpool, refarray, refvalue
-using GLM: Link, canonicallink, linkfun, linkinv, dispersion, dispersion_parameter
+using BSplineKit: BSplineKit
+using BSplineKit.BSplines: BSplineOrder
+using BSplineKit.BoundaryConditions: Natural
+using BSplineKit.DifferentialOps: Derivative
+using BSplineKit.SplineInterpolations: SplineInterpolation, interpolate
```

Comment:
can these be grabbed from the toplevel module (i.e. are they exported from the submodule to the toplevel?)?